.github/workflows/deploy.yml

```diff
@@ -4,38 +4,127 @@ on:
   release:
     types:
       - published
+
 jobs:
+  deploy_pre_release:
+    name: Deploy OpenAPI Generator Client Pre-Release to PYPI
+    if: ${{ github.event.release.prerelease && github.event.release.target_commitish == 'feature/openapi-generator-sdk' }}
+    runs-on: ubuntu-latest
+    env:
+      BW_ACCOUNT_ID: ${{ secrets.BW_ACCOUNT_ID }}
+      BW_USERNAME: ${{ secrets.BW_USERNAME }}
+      BW_PASSWORD: ${{ secrets.BW_PASSWORD }}
+      BW_USERNAME_FORBIDDEN: ${{ secrets.BW_USERNAME_FORBIDDEN }}
+      BW_PASSWORD_FORBIDDEN: ${{ secrets.BW_PASSWORD_FORBIDDEN }}
+      BW_VOICE_APPLICATION_ID: ${{ secrets.BW_VOICE_APPLICATION_ID }}
+      BW_MESSAGING_APPLICATION_ID: ${{ secrets.BW_MESSAGING_APPLICATION_ID }}
+      BW_NUMBER: ${{ secrets.BW_NUMBER }}
+      USER_NUMBER: ${{ secrets.USER_NUMBER }}
+      VZW_NUMBER: ${{ secrets.VZW_NUMBER }}
+      ATT_NUMBER: ${{ secrets.ATT_NUMBER }}
+      T_MOBILE_NUMBER: ${{ secrets.T_MOBILE_NUMBER }}
+      BASE_CALLBACK_URL: ${{ secrets.BASE_CALLBACK_URL }}
+      PYTHON_VERSION: ${{ matrix.python-version }}
+      OPERATING_SYSTEM: ${{ matrix.os }}
+      MANTECA_ACTIVE_NUMBER: ${{ secrets.MANTECA_ACTIVE_NUMBER }}
+      MANTECA_IDLE_NUMBER: ${{ secrets.MANTECA_IDLE_NUMBER }}
+      MANTECA_BASE_URL: ${{ secrets.MANTECA_BASE_URL }}
+      MANTECA_APPLICATION_ID: ${{ secrets.MANTECA_APPLICATION_ID }}
+      BW_NUMBER_PROVIDER: ${{ secrets.BW_NUMBER_PROVIDER }}
+    steps:
+      - name: Set Release Version
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/v}" >> $GITHUB_ENV
```

Comment:
Unable to grasp the syntax `${GITHUB_REF#refs/*/v}` and resolved value!

Reply:
Yeah, I'm not sure what that /v is for, and since this workflow runs on a release I think you should be table to just do either `${GITHUB_REF#refs/tags/}` or even just `${GITHUB_REF_NAME`} to get the release tag.

Reply:
@brianluisgomez @bpateldx  we use `v` in the release tag ex `v14.5.0` but that is invalid in the `setup.py` file. This grabs everything after the v in the release tag. 

This logic comes from our [beta deploy logic in the node SDK](https://github.com/Bandwidth/node-voice/blob/b7b2eec737857952a546b570a8803d85f9b01ae2/.github/workflows/deploy-beta.yml#L23) - where the npm CLI doesnt accept the `v` either

Reply:
@ajrice6713 gotcha, so I guess you can do it the way you have or include tags like `${GITHUB_REF#refs/tags/v}` if we wanted to take that wildcard out since this shouldn't get run on anything other than releases

Reply:
Yeah scoping it to tags is probably safer